README.md

```diff
@@ -237,6 +237,15 @@ Actions:
 * Update a deal - `client->deals->update`
 * Delete a deal - `client->deals->destroy`
 
+**Note about deal value**
+
+You can use either a string or numerical deal value when modifying a deal.
+```php
+$deal['value'] = 10;
+$deal['value'] = 10.10;
```

Comment:
We have decided (with @grzegorz-mysliwiec) that we want to use the same type that the client returns.